scripts/system/create/entityList/html/js/entityList.js

```diff
@@ -1536,6 +1565,32 @@ function loaded() {
                 return;
             }
 
+            if (controlKey && keyCodeString === "I") {
```

Comment:
Do you need to test that only the control key is pressed and not Alt and/or Shift?

Reply:
Hmm. I'm not sure what you mean.
This want to respond to Ctrl-i for "Inverse Selection"
As what was already done for "Ctrl-a" (for "Select All") at line 1545
Please give me additional detail about what you would expect to see adjusted.

Reply:
For example, test "if control key pressed but shift and alt keys aren't pressed".

Reply:
Got it... current Ctrl-A but also Ctrl-Shift-A, Ctrl-Alt-A and Ctrl-Shift-Alt-A were also doing the Select All

I will change the 2 conditions like this:
if (controlKey && !shiftKey && !altKey && keyCodeString === "A") {
if (controlKey && !shiftKey && !altKey && keyCodeString === "I") {
so only CTRL-A and CTRL-I will do it.